azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/Account.java

```diff
@@ -154,12 +148,17 @@ public void selectSubscription(List<String> selectedSubscriptionIds) {
         });
     }
 
+    public Mono<Account> continueLogin() {
+        Azure.az(AzureAccount.class).setAccount(this);
+        return Mono.just(this);
+    }
+
     private void finishLogin() {
```

Comment:
method name should better indicates what this method does, but not where this method is called.